app/src/main/java/dev/marcosfarias/pokedex/di/NetworkModule.kt

```diff
@@ -0,0 +1,21 @@
+package dev.marcosfarias.pokedex.di
+
+import dev.marcosfarias.pokedex.repository.PokemonService
+import org.koin.dsl.module
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+val networkModule = module {
+    single<Retrofit> {
+        Retrofit.Builder()
+            .baseUrl("https://gist.githubusercontent.com/mrcsxsiq/b94dbe9ab67147b642baa9109ce16e44/raw/97811a5df2df7304b5bc4fbb9ee018a0339b8a38/")
+            .addConverterFactory(GsonConverterFactory.create())
+            .build()
+    }
+
+    factory<PokemonService> {
+        get<Retrofit>().create(
```

Comment:
use the Kotlin extension function for create: `get<Retrofit>().create()`

See https://github.com/square/retrofit/blob/master/retrofit/src/main/java/retrofit2/KotlinExtensions.kt#L29